website/docs/d/db_subnet_group.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "aws"
+page_title: "AWS: aws_db_subnet_group"
+sidebar_current: "docs-aws-datasource-db-subnet-group"
```

Comment:
i believe this is no longer used and can be omitted...looks like the `awsproviderlinter` may check for these